### PR TITLE
fix #79

### LINKED ISSUE #79: Not feedback when elem is null in the browser
when in a web browser the `elem` is still not loaded [this line](https://github.com/Raynos/mercury/blob/e480a8aa0f758f8bf4657c8375333c0d8031bcc5/index.js#L54) occults that error  and don't give any type of feedback

this can be painful for newbies

### PR DESCRIPTION
**Pull request by klemola**
Fixed issue #79 

Added a test for the new functionality. I had to disable jscs disallowKeywords feature for the test to use try/catch.

I also added some Intellij Idea specific lines to .gitignore.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -15,6 +15,19 @@ test('mercury is a object', function t(assert) {
     assert.end();
 });
 
+// jscs:disable disallowKeywords
+test('missing element prevents app init', function t(assert) {
+    assert.throws(mercury);
+
+    try {
```
Comment (Raynos) [inline test/index.js]: You can also try assert.throws

File: index.js [hunk 2]
```diff
@@ -107,14 +107,20 @@ function channels(funcs, context) {
 }
 
 function app(elem, observ, render, opts) {
+    if (!elem) {
+        throw new ReferenceError(
```
Comment (Raynos) [inline index.js]: Let's not use reference error as that is meant for the language. We can just use Error.

**Comment by klemola** [comment]
Thanks for the comments! I added your proposed changes to the code.

**Comment by Raynos** [comment]
lgtm.

**Pull request merged**